### DISCUSSION TITLE
Beyond gender   [Zhao Mingxuan, Shi Jian]

### DISCUSSION THREAD

**Discussion by TILLA0070** [realitydeslab/playshop-summer-2024#23] [Proposal]
Beyond gender:性别之上

Team:Zhao Mingxuan, Shi Jian

Research Questions:

What are the implications of gender trends under the influence of political forces? As the scope of gender identity is broadened, will gender be blurred out and become less important? How will our perspective on gender change?

Keywords:
Gender discrimination, ai system, gender dualism

Artefact / Media:
Technical news, websites，Interactive imaging，video

Abstract:
This paper presents a speculative line design based on the potential space of gender.Beyond Gender is a soon-to-be-released daily gender testing system with an eye on the near future. By reflecting the rise of the LGBT movement and its control in the context of Western political correctness through everyday gender testing, it leads people to look back at themselves and re-examine gender issues.Beyongd Gender allows everyone to take a variety of gender tests in which different gender images are derived from personal information provided by the user, which will be an interesting experience in relation to "gender divination". It will be an interesting experience of "gender divination". This paper opens up a discussion on the socio-political forces that shape our perceptions of gender, raising critical questions about the intersection of gender identity, political correctness, and social norms, and examining them in the context of speculative design.

Reading list
http://dl.acm.g.yyttgd.top/doi/10.1145/3274357The Misgendering Machines: Trans/HCI Implications of Automatic Gender Recognition
性别歧视机器：自动性别识别对跨性别/人机交互的影响
http://dl.acm.g.yyttgd.top/doi/10.1145/3605098.3635935How Facial Recognition Technologies Affect the Transgender Community? A Systematic Mapping Study
面部识别技术如何影响跨性别社区？系统测绘研究

Visual reference

https://www.youtube.com/watch?v=kSLJriaOumA

https://github.com/realitydeslab/playshop-summer-2024/assets/134136563/4f94f8ad-2411-4c68-ab10-37c4e5d33672

https://www.youtube.com/watch?v=YxmqUIBuWbM

https://github.com/realitydeslab/playshop-summer-2024/assets/134136563/90859094-f02b-4215-8842-435938c4e43d

Reference：
Freud, S. (1923). The Ego and the Id. London: Hogarth Press.
Bem, S. L. (1974). The Measurement of Psychological Androgyny. Journal of Consulting and Clinical Psychology, 42(2), 155-162.
Bettcher, T. M. (2007). Evil Deceivers and Make-Believers: On Transphobic Violence and the Politics of Illusion. Hypatia, 22(3), 43-65.
Stryker, S. (2006). (De)Subjugated Knowledges: An Introduction to Transgender Studies. In Susan Stryker & Stephen Whittle (Eds.), The Transgender Studies Reader (pp. 1-17). New York: Routledge.
D'Emilio, J. (1998). Sexual Politics, Sexual Communities. Chicago: University of Chicago Press.
Cohen, C. J. (2005). Punks, Bulldaggers, and Welfare Queens: The Radical Potential of Queer Politics? In Linda Nicholson & Nancy Fraser (Eds.), Socialist Feminism: Out of the Blue? (pp. 57-76). New York: Routledge.
Rupp, L. J., & Taylor, V. (2003). Drag Queens at the 801 Cabaret. Chicago: University of Chicago Press.
Hill, M. E. (2009). Whiteness as Property: A Legal History. In George Lipsitz (Ed.), The Possessive Investment in Whiteness: How White People Profit from Identity Politics (pp. 107-133). Philadelphia: Temple University Press.
Bauer, G. R. (2014). Incorporating Intersectionality Theory into Population Health Research Methodology: Challenges and the Potential to Advance Health Equity. Social Science & Medicine, 101, 10-17.
Dunne, A., & Raby, F. (2013). Speculative Everything: Design, Fiction, and Social Dreaming. Cambridge, MA: MIT Press.
Lupton, E. (2013). The Design of Everyday Things: Revised and Expanded Edition. Indianapolis, IN: Quezon City Press.
Gunn, W., & Lupton, E. (2013). Designing Qualitative Research. Thousand Oaks, CA: SAGE Publications.

**Comment by TILLA0070** [comment]
![微信图片_20240626152222](https://github.com/realitydeslab/playshop-summer-2024/assets/134136563/061a9204-e907-457b-8e47-a72770f8a263)

**Comment by botaohu** [comment]
https://www.lesswrong.com/posts/jnoJac2wzexmfKxmN/gender-vectors-in-rome-s-latent-space

**Comment by TILLA0070** [comment]
https://github.com/orpatashnik/StyleCLIP

**Reply by TILLA0070** [reply]
![0016-0038 00_00_00-00_00_30](https://github.com/realitydeslab/playshop-summer-2024/assets/134136563/c85a27ac-dc35-41f3-b3cb-8b32d772c0f0)

**Comment by glitch-kraft** [comment]
2 axis exercise :
Dualism of Gender vs Pluralism of Gender
Tribalism in Politics vs Plurality in Politics

**Comment by TILLA0070** [comment]
Key frame
![keyframe](https://github.com/realitydeslab/playshop-summer-2024/assets/134136563/6371a3d9-87a5-4cfc-86a7-4d2c096fd88e)

**Reply by TILLA0070** [reply]
tagline: when gender becomes everything,then everything becomes gender.don't forget the elephant in the room,who swallowed in the darkness,in the lantent space.

**Reply by TILLA0070** [reply]
prompt:In the parade,A man turned into a goat, then into a tea cup wearing a wedding dress, and finally into a neat punk robot.

**Comment by TILLA0070** [comment]
Circular argument may trapped you.When  gender becomes everything,then everything becomes  gender.Don't forget the elephant in the room,who swallowed in the darkness,in the lantent space.

**Reply by glitch-kraft** [reply]
When gender becomes everything, everything becomes gender. How an Ai latent space can help us unpack and define the diversity of what gender can be.
